client/src/screens/BallotScreen.tsx

```diff
@@ -200,12 +201,41 @@ const BallotScreen = () => {
                   <li key={candidate.id}>{candidate.name}</li>
                 ))}
               </ul>
-            ) : (
-              <ul>
-                <li>Yes</li>
-                <li>No</li>
-              </ul>
-            )}
+            ) : null}
+            {contest.type === 'yesno' ? (
+              <React.Fragment>
+                <Text
+                  preLine
+                  dangerouslySetInnerHTML={{
+                    __html: DOMPurify.sanitize(contest.description),
+                  }}
+                />
+                <ul>
+                  <li>Yes</li>
+                  <li>No</li>
+                </ul>
+              </React.Fragment>
+            ) : null}
```

Comment:
Any reason you're using a ternary vs expression:
```
{contest.type === 'yesno' && ( ... )}
```

Reply:
no good reason, was just mimicking previous code structure in a rush.

Reply:
If you were to combine the ternaries you get type refinement as you go:

```ts
contest.type === 'candidate'
  ? // must be candidate contest
  : // could be yesno or ms-either-neither
  contest.type == 'yesno'
  ? // must be yesno
  : // must be ms-either-neither
```

Reply:
I'm gonna make myself a new t-shirt:

I ❤️ nested terinaries